platform/admin/admin-configuration-passwordencryption/src/main/java/org/codice/ddf/admin/configuration/password/encryption/PasswordEncryptingPersistenceManager.java

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.admin.configuration.password.encryption;
+
+import java.io.IOException;
+import java.security.PrivilegedActionException;
+import java.util.Dictionary;
+import java.util.Enumeration;
+import java.util.Hashtable;
+import java.util.List;
+import java.util.function.Function;
+
+import org.apache.felix.cm.PersistenceManager;
+import org.codice.ddf.admin.core.api.ConfigurationAdmin;
+import org.codice.ddf.admin.core.api.Metatype;
+import org.codice.ddf.admin.core.api.MetatypeAttribute;
+import org.codice.ddf.admin.core.api.Service;
+import org.codice.ddf.security.common.Security;
+import org.osgi.service.metatype.AttributeDefinition;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ddf.security.encryption.EncryptionService;
+
+public class PasswordEncryptingPersistenceManager implements PersistenceManager {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(
+            PasswordEncryptingPersistenceManager.class);
+
+    private PersistenceManager persistenceManager;
+
+    //findbugs incorrectly flagged this as an unread field and there was no way to skip it with the annotations that worked
+    //findbugs is thus disabled for this class
+    private EncryptionService encryptionService;
```

Comment:
That was my assumption as well. I tried everything I could think of to fix it without changing my code (because I like my code) or to just skip the single issue that it was flagging, but nothing worked.